client/src/app/slides/assignments/poll/poll-slide-data.ts

```diff
@@ -0,0 +1,15 @@
+export interface PollSlideData {
+    title: string;
+    assignments_poll_100_percent_base: string; // TODO: The type is somewhere defined, isn't it?
+    poll: {
+        published: boolean;
+        description?: string;
+        has_votes?: boolean;
+        pollmethod?: string; // TODO: The type is somewhere defined, isn't it?
+        votesno?: string; // TODO: same conversion needed as for the PollModel
```

Comment:
Are these (no, abstain on the per-poll level) even used anywhere?

Reply:
I do not know exactly. With the voting plugin they were showed. I think, we should show them, if they have any value.